cwf/cloud/go/services/cwf/analytics/calculations/Authentications.go

```diff
@@ -14,7 +14,7 @@ import (
 // AuthenticationsCalculation holds the parameters needed to run an authentication
 // query and the registered prometheus gauge that the resulting value should be stored in
 type AuthenticationsCalculation struct {
-	calculations.CalculationParams
+	calculations.BaseCalculation
```

Comment:
Rename Authentications.go and XAP.go to authentications.go and xap.go

Reply:
I had already done the renaming. Not sure why it isn’t merged that way